src/control/tools/StrokeStabilizer.h

```diff
@@ -0,0 +1,666 @@
+/*
+ * Xournal++
+ *
+ * Handles input of strokes
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+#pragma once
+
+#include <array>
+#include <cmath>
+#include <deque>
+#include <functional>
+
+// #include <boost/circular_buffer.hpp>
```

Comment:
In term, sure. Still, someone may have a strong opinion on my in-house half-baked `CircularBuffer`. I'll just keep this conversation open for now (so that people may argue for boost's come back).

Reply:
> In term, sure. Still, someone may have a strong opinion on my in-house half-baked CircularBuffer

Okay! That makes sense!